services/UserAccountService.ts

```diff
@@ -234,7 +234,6 @@ export default class UserAccountService {
 
         const updatedUser = await userRepository.upsertUser(currUser, { accessType });
 
-        // log the activity of changing a user's access type
```

Comment:
I will be honest this specific comment is fine as it explains a decent size of code